packages/contracts/test/token/erc20/governance-wrapped-erc20.ts

```diff
@@ -36,12 +38,13 @@ describe('GovernanceWrappedERC20', function () {
   let signers: SignerWithAddress[];
   let governanceToken: GovernanceWrappedERC20;
   let erc20: TestERC20;
+  let dao: DAO;
   let TestERC20: TestERC20__factory;
   let GovernanceWrappedERC20: GovernanceWrappedERC20__factory;
   let defaultBalances: AccountBalance[];
 
   let defaultExistingERC20InitData: [string, string, number];
-  let defaultGovernanceWrappedERC20InitData: [string, string, string];
+  let defaultGovernanceWrappedERC20InitData: any[] = [];
```

Comment:
Previously, this was more type-safe. Please revert

Reply:
https://github.com/aragon/osx/pull/596/commits/6710f4efa85095b3a2451c25180643479940cb66